app/api/__init__.py

```diff
@@ -69,3 +73,9 @@
 api.route(TrackDetail, 'track_detail', '/tracks/<int:id>', '/sessions/<int:session_id>/track')
 api.route(TrackRelationship, 'track_sessions', '/tracks/<int:id>/relationships/sessions')
 api.route(TrackRelationship, 'track_event', '/tracks/<int:id>/relationships/event')
+
+# session_types
+api.route(SessionTypeList, 'session_type_list', '/session_types', '/events/<int:event_id>/session_types')
```

Comment:
Changes done. The endpoints remaining are 
- /events/<int:id>/relationships/session_types
- /sessions/<int:id>/relationships/session_type
- /events/<int:event_id>/session_types
- /sessions/<int:session_id>/session_type
Is that correct?